### ISSUE TITLE
<div>*</div>

### ISSUE THREAD

**Issue by b4z81** [jacobrask/styledocco#8]
this code 

``` html
<div>*</div>
<div>*</div>

```

 in the css comment will generated a bad html..

``` html
       <div class="cols-12">
        <label for=""><em class="form-req">*</em></label>
        <textarea></textarea> 
      </div>
      <div class="cols-12">
        <label for="">Msg <em class="form-req">*</em></label>
        <textarea></textarea> 
      </div>
```

**Comment by jacobrask** [comment]
Hi, thank you for your report. `*` in code outputs `<em>` for some reason, it has to do with the Markdown parsing - but that should not be enabled for code snippets. I added a test and will investigate further.

**Comment by jacobrask** [comment]
I just released version 0.2.2 with a fix for this.

**Comment by b4z81** [comment]
great job, thank you!
